docs/sources/tutorials/alerting-get-started-pt2/index.md

```diff
@@ -0,0 +1,172 @@
+
+In part 2, we dig into more complex yet equally fundamental elements of Grafana Alerting: **alert instances** and **notification policies**.
+
+After introducing each component, you will learn how to:
+
+- Configure an alert rule that returns more than one alert instance
+- Create notification policies that route firing alert instances to different contact points
+- Use labels to match alert instances and notification policies
+
+Learning about alert instances and notification policies is useful if you have more than one contact point in your organization, or if your alert rule returns a number of metrics that you want to handle separately by routing each alert instance to a specific contact point. The tutorial will introduce each concept, followed by how to apply both concepts in a real-world scenario.
+
+## Alert instances
+
+An [alert instance](https://grafana.com/docs/grafana/latest/alerting/fundamentals/#alert-instances) is an event that matches a metric returned by a query of an alert rule.
+
+Let's consider a scenario where you're monitoring website traffic using Grafana. You've set up an alert rule to trigger an alert instance if the number of page views exceeds a certain threshold (more than `1000` page views) within a specific time period, say, over the past `5` minutes.
+
+If the query returns more than one time-series, each time-series represents a different metric or aspect being monitored. In this case, the alert rule is applied individually to each time-series.
+
+{{< figure src="/media/docs/alerting/get-started-digram-instance-grey.png" max-width="1200px" caption="Alert Instances in the Context of an Alert Rule" >}}
+
+In this scenario, each time-series is evaluated independently against the alert rule. It results in the creation of an alert instance for each time-series. The time-series corresponding to the desktop page views meets the threshold and, therefore, results in an alert instance in **Firing** state for which an alert notification will be sent. The mobile alert instance state remains **Normal**.
+
+## Notification policies
+
+[Notification policies](https://grafana.com/docs/grafana/latest/alerting/fundamentals/notifications/notification-policies/) route alerts to different communication channels, reducing alert noise and providing control over when and how alerts are sent. For example, you might use notification policies to ensure that critical alerts about server downtime are sent immediately to the on-call engineer. Another use case could be routing performance alerts to the development team for review and action.
+
+Key Characteristics:
+
+- Route alert notifications by matching alerts and policies with labels
+- Manage when to send notifications
+
+{{< figure src="/media/docs/alerting/get-started-notification-policy-tree-combo.png" max-width="1200px" caption="Routing of alerts with notification policies" >}}
```

Comment:
re accessibility - every image should have alt text: {{< figure alt="<ALT TEXT>"  src="/media/<PATH TO IMAGE>" >}}